libvast/vast/system/terminator.hpp

```diff
@@ -34,7 +35,14 @@ struct terminator_state {
 };
 
 /// Performs a parallel shutdown of a list of actors.
+/// @param self That terminator actor.
```

Comment:
"That"?

Reply:
Yes, *that* pesky actor. 😅 

(Will change it.)